### PR TITLE
[Extension] Add Info Icon on Export Design Modal

### PR DESCRIPTION
**Pull request by AjaxAueleke**
**Notes for Reviewers**

This PR adds an info icon on export design modal.

I was asked to update the modal so that it will have an info icon with a link to Exporting a Design page [(this)](https://docs.layer5.io/meshmap/designer/export-designs/) on Meshery Doc.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by AjaxAueleke** [comment]
@iArchitSharma

**Comment by iArchitSharma** [comment]
Can you please add screenshot??

File: ui/components/ExportModal.js [hunk 1]
```diff
@@ -218,6 +243,18 @@ const ExportModal = (props) => {
           {ExtensibleButton && <ExtensibleButton {...props} closed={closed} />}
         </Box>
       </DialogContent>
+      <DialogActions>
+        <CustomTextTooltip
+          backgroundColor="#3C494F"
+          placement="top"
+          interactive={true}
+          title={renderTooltipContent()}
```
Comment (Yashsharma1911) [inline ui/components/ExportModal.js:251]: Can you use this function which already has a predefined regex to identify a link 

https://github.com/layer5labs/meshery-extensions/blob/64d2f06b58d7f4795501f57e54e03cef6770149d/meshmap/src/sections/MesheryDesignerComponent/PatternServiceFormWrapper/RJSFCustomComponents/utils.js#L56

File: ui/components/ExportModal.js [hunk 2]
```diff
@@ -74,6 +78,10 @@ const styles = (theme) => ({
       },
     },
   },
+  toolTip: {
+    textDecoration: 'underline',
+    color: theme.palette.secondary.link2,
```
Comment (Yashsharma1911) [inline ui/components/ExportModal.js:83]: Nice work by adding theme color

File: ui/components/ExportModal.js (hunk 1 above)
Comment (Yashsharma1911) [reply ui/components/ExportModal.js:251]: I missed, you raised this PR in Meshery :) however we do have same function in Meshery too I guess you updated that right?

**Comment by AjaxAueleke** [comment]
@Yashsharma1911 please review

File: ui/components/ExportModal.js [hunk 3]
```diff
@@ -84,6 +105,12 @@ const ExportModal = (props) => {
     classes,
     ExtensibleButton,
   } = props;
+  const renderTooltipContent = () => {
+    return getHyperLinkDiv(
+      'MeshMap Designer offers multiple export options, allowing you to choose the format that suits your needs. https://docs.layer5.io/meshmap/designer/export-designs/',
+      'Learn more',
```
Comment (Yashsharma1911) [inline ui/components/ExportModal.js:111]: @AjaxAueleke thankyou however we discuss to have support of passing link something like this 
```suggestion
      'MeshMap Designer offers multiple export options, allowing you to choose the format that suits your needs. [Learn More](https://docs.layer5.io/meshmap/designer/export-designs/)'
```

Can you make sure we are passing link in this way

**Review by Yashsharma1911 — CHANGES_REQUESTED** [review]

File: ui/components/ExportModal.js [hunk 4]
```diff
@@ -14,6 +14,10 @@ import { GetApp as GetAppIcon } from '@material-ui/icons';
 import OriginalApplicationFileIcon from '@/assets/icons/OriginalApplicationIcon';
 import ModifiedApplicationFileIcon from '@/assets/icons/ModifiedApplicationIcon';
 import { withStyles } from '@material-ui/core/styles';
+import Link from 'next/link';
+import { CustomTextTooltip } from '@/components/MesheryMeshInterface/PatternService/CustomTextTooltip';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/ExportModal.js]: @nebula-aac, docile you link to this icon in Sistent, so that @AjaxAueleke might use from the central library?

**Comment by leecalcote** [comment]
I accepted @Yashsharma1911’s suggested change.

File: ui/components/ExportModal.js [hunk 5]
```diff
@@ -84,6 +105,11 @@ const ExportModal = (props) => {
     classes,
     ExtensibleButton,
   } = props;
+  const renderTooltipContent = () => {
+    return getHyperLinkDiv(
+      'MeshMap Designer offers multiple export options, allowing you to choose the format that suits your needs. [Learn More](https://docs.layer5.io/meshmap/designer/export-designs/)'
```
Comment (leecalcote) [inline ui/components/ExportModal.js:110]: Lovely.

File: ui/components/ExportModal.js [hunk 6]
```diff
@@ -218,6 +244,18 @@ const ExportModal = (props) => {
           {ExtensibleButton && <ExtensibleButton {...props} closed={closed} />}
         </Box>
       </DialogContent>
+      <DialogActions className={classes.dialogAction}>
+        <CustomTextTooltip
+          backgroundColor="#3C494F"
```
Comment (leecalcote) [inline ui/components/ExportModal.js:249]: theme reference for color...?

Comment (AjaxAueleke) [reply ui/components/ExportModal.js]: fixed

File: ui/components/MesheryMeshInterface/PatternService/helper.js
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/helper.js:1]: Ok, sounds great.

**Review by leecalcote — APPROVED** [review]
Appears to be ready...

**Comment by leecalcote** [comment]
@Yashsharma1911 is this one good to go?

**Review by Yashsharma1911 — APPROVED** [review]
LGTM

**Comment by Yashsharma1911** [comment]
Ohh this was your first PR in Meshery, nice 👏

**Pull request merged**